src/NuGet.Core/NuGet.Configuration/Utility/SettingsUtility.cs

```diff
@@ -11,7 +11,7 @@ public static class SettingsUtility
         public const string ConfigSection = "config";
         public const string GlobalPackagesFolderKey = "globalPackagesFolder";
         public const string GlobalPackagesFolderEnvironmentKey = "NUGET_PACKAGES";
-        public const string DefaultGlobalPackagesFolderPath = ".nuget\\packages\\";
+        public const string DefaultGlobalPackagesFolderPath = ".nuget/packages/";
```

Comment:
Can you explain why this change is necessary, rather than do the replace below in the opposite direction?

I would instead probably use a path.combine to generate a readonly string on the fly?